<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Colors and Widths</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>343</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QPushButton" name="XsColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="RsCheckBox">
      <property name="text">
       <string>Rs</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="RsColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="XsLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="Gain1CheckBox">
      <property name="text">
       <string>Gain1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="RLColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="ZLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="Z0ColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="ZCheckBox">
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="RsLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="AngleLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="AngleColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="XsCheckBox">
      <property name="text">
       <string>Xs</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QSpinBox" name="Z0LineSpinBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="SWRColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="Z0CheckBox">
      <property name="text">
       <string>50om</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="SWRLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="SWRCheckBox">
      <property name="text">
       <string>SWR</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="ZColorPushButton_2">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="AngleCheckBox">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="RLCheckBox">
      <property name="text">
       <string>R. L.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="RLLineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="Gain2CheckBox">
      <property name="text">
       <string>Gain2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="Gain3CheckBox">
      <property name="text">
       <string>Gain3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Gain1ColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Gain2ColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="Gain3ColorPushButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="Gain1LineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="Gain2LineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QSpinBox" name="Gain3LineSpinBox">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ClosePushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
